corehq/apps/auditcare/utils/export.py

```diff
@@ -70,6 +68,48 @@ def get_all_log_events(start_date=None, end_date=None):
     )
 
 
+def get_domain_first_access_times(domains, start_date=None, end_date=None):
+    """Query NavigationEventAudit events for _first event matching any of
+    `domains` within each authenticated session_.
+
+    NOTE: This function does _not_ query couch.
+
+    NOTE: This function may return multiple "access events" from the same
+          session (if multiple of `domains` were accessed in the same session).
+
+    Resulting SQL query:
+
+    ```sql
+    SELECT
+        MIN(event_date) AS access_time,
+        "user",
+        domain,
+        session_key
+    FROM auditcare_navigationeventaudit
+    WHERE (
+        domain IN ( {domains} )
+        AND event_date > {start_date}
+        AND event_date < {end_date}
+        AND "user" IS NOT NULL
+        AND session_key IS NOT NULL
+    )
+    GROUP BY ("user", domain, session_key)
+    ORDER BY access_time ASC;
+    ```
+    """
+    sql_start_date = determine_sql_start_date(start_date)
+    where = get_date_range_where(sql_start_date, end_date)
+    where["domain__in"] = domains
+    where["user__isnull"] = False
+    where["session_key__isnull"] = False
+    return (NavigationEventAudit.objects
+            .values("user", "domain", "session_key")
```

Comment:
I think our log exports should basically never include `session_key` because of the sensitivity to that field. While expired session keys may be essentially harmless, non-expired ones are indeed very sensitive, and I'd prefer if the person running this export didn't have to remember to carefully think about the difference each time.

Reply:
Oh, sorry, I see in the place you actually write this out, `session_key` is not included. That's good. Can you just confirm that's the case? I think then this comment just becomes, if we're not using it do we need to return it from this function?

Reply:
Maybe it's for tests. If all that checks out, I'm fine leaving it as it is. Sorry for the rollercoster as I talked myself through it 🎢

Reply:
Thanks for the critical eye. Your second observation is correct, the `session_key` value is _not_ exported.  Its presence in the `values()` QuerySet method is used define the `GROUP BY` clause, with the unfortunately side-effect of implicitly including it in the `SELECT` clause of the underlying query.  I didn't dig deeper into the ORM to determine if/how to explicitly exclude that column from the `SELECT` clause.  I will have another look because your idea about the backend not returning this value is a very good one.

The column is also not needed for any tests.

Reply:
Thanks for this suggestion. I've updated the query to not include the `session_key` in the result-set.